charts/featureform/templates/streamer_proxy/deployment.yaml

```diff
@@ -0,0 +1,38 @@
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+#  Copyright 2024 FeatureForm Inc.
+#
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: iceberg-proxy
+spec:
+  {{- if not .Values.icebergProxy.autoscaling.enabled }}
+  replicas: {{ .Values.icebergProxy.replicaCount }}
+  {{- end }}
+  selector:
+    matchLabels:
+      app: iceberg-proxy
+  template:
+    metadata:
+      labels:
+        app: iceberg-proxy
+      {{ if .Values.restartOnUpdate }}
+      annotations:
+        timestamp: {{ now | quote }}
+      {{ end }}
+    spec:
+      imagePullSecrets:
+        - name: {{ .Values.k8sImagePullSecret }}
+      containers:
+      - name: iceberg-proxy
+        image: "{{ .Values.repository | default .Values.repository }}/{{ .Values.icebergProxy.image.name }}:{{ .Values.versionOverride | default .Chart.AppVersion }}"
```

Comment:
`{{ .Values.repository | default .Values.repository }}` is a no-op to default to `Values.repository`. I notice this issue with a few other templates, can bring up a separate PR to fix the other ones.

Reply:
good eye. too much copy-pasta 🍝